### PR TITLE
add SGS STAS-PLUS for OCM-Online Model

### PR DESCRIPTION
**Pull request by Wha1eFa11**
### Thank you for contributing to the IoT Plug and Play Models repository

:memo: Please review this checklist before submission

- [x] :eyes: The models submitted in this PR follow the [DTDL Naming Guidelines](https://github.com/Azure/iot-plugandplay-models-tools/wiki/DTDL-Naming-Guidelines) for contributing to this repository.
- [x] 🕵️I have validated the models locally using the [dmr-client](https://github.com/Azure/iot-plugandplay-models-tools/tree/dev/clients/dotnet/Microsoft.IoT.ModelsRepository.CommandLine) command line tool.
- [x] ❗ I understand once the PR is approved, the models are considered immutable: any edits will require to submit new files.
- [x] ✨ When creating a new top level `dtmi:namespace`, I have completed the [PR Info Template](#pr-info-template) below.

:zap: PR validation steps are described in the tools [Wiki](https://github.com/Azure/iot-plugandplay-models-tools/wiki/Validation-Pipeline#pr-validation-checks).

## PR Info Template

When submitting models to the repository we ask that you provide as much of the following meta information around your models and related devices as possible. This info will be used to improve the _IoT Plug and Play_ ecosystem.

👇: Please replace the markdown comment examples with your own values.

### Company Info

>GuangZhou ShineTech Intelligent Technology Co., Ltd. .

### Model Submission Goals

> Device certification.  

### Code Owners & Reserved Names

> Indicates GitHub alias entries to be added to the repo `CODEOWNERS` for the incoming model namespace. The codeowner is expected to be involved in subsequent DTDL model submissions against the same namespace.

This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

### CODE REVIEW

File: dtmi/sgs/ocm_online-2.json [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+      "name": "Flow_Sensor",
+      "displayName": "Sensor flow",
+      "description": "Flow of metal abrasive particle sensor",
+      "schema": "float",
+      "unit": "ml/min"
+    },
+    {
+      "@type": [
+        "Property",
+        "Concentration"
```
Comment (jrdouceur) [inline dtmi/sgs/ocm_online-2.json:247]: Although DTDL v2 does not have an appropriate semantic type, DTDL v3 uses "Concentration" for unitless quantities such as percent and parts per million, and it uses "Density" for mass/volume ratios.  For forward-compatibility, Density would be preferable to Concentration here.

File: dtmi/sgs/ocm_online-2.json [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+      ],
+      "name": "FeConcentration",
+      "displayName": "Fe Concentration",
+      "description": "Ferromagnetic concentration",
+      "schema": "float",
+      "unit": "mg/L"
```
Comment (jrdouceur) [inline dtmi/sgs/ocm_online-2.json:253]: For actual semantic types, the "unit" property expects a JSON-LD term that maps to an IRI, so "mg/L" is not syntactically valid.  For forward compatibility, "milligramPerLitre" would be preferable to "mg/L".

File: dtmi/sgs/ocm_online-2.json [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+      ],
+      "name": "NonFeTotal24",
+      "displayName": "NonFeTotal24",
+      "description": "Total number of Non ferromagnetic particles in 24 hours",
+      "schema": "integer"
+    },
+    {
+      "@type": [
+        "Property",
+        "Flow"
```
Comment (jrdouceur) [inline dtmi/sgs/ocm_online-2.json:236]: Although DTDL v2 does not have an appropriate semantic type, DTDL v3 uses "VolumeFlowRate" for this.  For forward-compatibility, VolumeFlowRate would be preferable to Flow.

File: dtmi/sgs/ocm_online-2.json [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+      ],
+      "name": "Flow_Sensor",
+      "displayName": "Sensor flow",
+      "description": "Flow of metal abrasive particle sensor",
+      "schema": "float",
+      "unit": "ml/min"
```
Comment (jrdouceur) [inline dtmi/sgs/ocm_online-2.json:242]: For actual semantic types, the "unit" property expects a JSON-LD term that maps to an IRI, so "ml/min" is not syntactically valid. For forward compatibility, "millilitrePerMinute" would be preferable to "ml/min".

File: dtmi/sgs/ocm_online-2.json [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+    {
+      "@type": [
+        "Property",
+        "Grade"
+      ],
+      "name": "Grade4um",
+      "displayName": ">4um Grade",
+      "description": "Particle grade greater than 4um",
+      "schema": "integer",
+      "unit": "Grade"
```
Comment (jrdouceur) [inline dtmi/sgs/ocm_online-2.json:17]: "Grade" is not really a unit, is it?  What does the integer number of this Property mean?

File: dtmi/sgs/ocm_online-2.json [hunk 6]
```diff
@@ -0,0 +1,275 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:SGS:OCM_Online;2",
+  "@type": "Interface",
+  "displayName": "OCM-Online",
+  "description": "Report the current oil quality data and oil iron content distribution data, and Get Log for STAS-PLUS",
```
Comment (jrdouceur) [inline dtmi/sgs/ocm_online-2.json:6]: description should end with a period

Comment (jrdouceur) [reply dtmi/sgs/ocm_online-2.json:6]: Here and below.

File: dtmi/sgs/ocm_online-2.json [hunk 7]
```diff
@@ -0,0 +1,275 @@
...
+  "contents": [
+    {
+      "@type": [
+        "Property",
+        "Grade"
+      ],
+      "name": "Grade4um",
```
Comment (jrdouceur) [inline dtmi/sgs/ocm_online-2.json:13]: name should be camelCase, here and below

**Review by rido-min — CHANGES_REQUESTED** [review]
Can you update the PR info template?

File: dtmi/sgs/ocm_online-2.json (hunk 1 above)
Comment (Wha1eFa11) [reply dtmi/sgs/ocm_online-2.json]: I modified it

File: dtmi/sgs/ocm_online-2.json (hunk 2 above)
Comment (Wha1eFa11) [reply dtmi/sgs/ocm_online-2.json]: I modified it

File: dtmi/sgs/ocm_online-2.json (hunk 4 above)
Comment (Wha1eFa11) [reply dtmi/sgs/ocm_online-2.json]: I modified it

File: dtmi/sgs/ocm_online-2.json (hunk 5 above)
Comment (Wha1eFa11) [reply dtmi/sgs/ocm_online-2.json]: I modified it

File: dtmi/sgs/ocm_online-2.json (hunk 6 above)
Comment (Wha1eFa11) [reply dtmi/sgs/ocm_online-2.json]: I modified it

File: dtmi/sgs/ocm_online-2.json (hunk 7 above)
Comment (Wha1eFa11) [reply dtmi/sgs/ocm_online-2.json]: I modified it

**Comment by Wha1eFa11** [comment]
> Can you update the PR info template?
-- I updated it

**Review by jrdouceur — APPROVED** [review]

**Review by rido-min — APPROVED** [review]

**Pull request merged**